indra/assemblers/pybel/assembler.py

```diff
@@ -1,19 +1,21 @@
 from __future__ import absolute_import, print_function, unicode_literals
-from builtins import dict, str
-import uuid
+
 import logging
+import uuid
+from copy import copy, deepcopy
+
 import networkx as nx
-from copy import deepcopy, copy
+from builtins import dict, str
+
 import pybel
 import pybel.constants as pc
+from indra.databases import hgnc_client
+from indra.statements import *
```

Comment:
Again, why move these? Typically indra imports should be the last in the order of things according to the standard.